java/opendcs/src/main/java/opendcs/dao/TsGroupDAO.java

```diff
@@ -460,6 +460,8 @@ public void deleteTsGroup(DbKey groupId)
 
 		q = "DELETE from tsdb_group WHERE group_id = " + groupId;
 		doModify(q);
+
+		cache.remove(groupId);
```

Comment:
I really want to pressure you to move all those queries into a transaction, but I'm about to merge the big change anyways and I suspect it would make more work.